Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/Allocator/Jobs/Mandate/Notification.hs

```diff
@@ -125,20 +140,18 @@ getRescheduledTime tc = addUTCTime tc.mandateNotificationRescheduleInterval <$>
 
 scheduleJobs :: (CacheFlow m r, EsqDBFlow m r, HasField "schedulerSetName" r Text, HasField "schedulerType" r SchedulerType, HasField "jobInfoMap" r (M.Map Text Bool)) => TransporterConfig -> UTCTime -> UTCTime -> Id Merchant -> Id MerchantOperatingCity -> Int -> m ()
 scheduleJobs transporterConfig startTime endTime merchantId merchantOpCityId maxShards = do
-  now <- getLocalCurrentTime transporterConfig.timeDiffFromUtc
   let dfExecutionTime = transporterConfig.driverAutoPayExecutionTime
-      dfNotificationTime = transporterConfig.driverAutoPayNotificationTime
       dfStatusCheckTime = transporterConfig.orderAndNotificationStatusCheckTime
-  let dfCalculationJobTs = diffUTCTime (addUTCTime (dfExecutionTime + dfNotificationTime) endTime) now
-  let orderAndNotiifcationJobTs = diffUTCTime (addUTCTime (dfStatusCheckTime + dfNotificationTime) endTime) now
+  let dfCalculationJobTs = dfExecutionTime
```

Comment:
It is not correct, right?

Reply:
When notification job ends from then + executionTimeGap It won't cause a issue If We are going dynamically.
A better approach I can think of is :- 
```
if (addUTCTime (dfStatusCheckTime + dfNotificationTime) endTime < now ()) then dfExecutionTime else diffUTCTime (addUTCTime (dfExecutionTime + dfNotificationTime) endTime) now
```
Similarly for order status